extensions/github/src/publish.ts

```diff
@@ -66,11 +58,49 @@ export async function publishRepository(gitAPI: GitAPI, repository?: Repository)
 		owner = user.data.login;
 	} catch (e) {
 		// User has cancelled sign in
-		quickpick.dispose();
 		return;
 	}
 
-	quickpick.busy = false;
+	let orgChoice: string = owner;
+	// TODO: While this api call is made, there isn't anything displayed to the
+	// user to show progress.  Showing either picker first risks having to swap
+	// them after the api call is finished and possibly causing confusion.
+	const orgs = await octokit.orgs.listForAuthenticatedUser();
+	// Only show the organization picker, if the user has an organization.
+	if (orgs.data.length) {
+		const orgQuickPick = vscode.window.createQuickPick<vscode.QuickPickItem & { org: string }>();
+		orgQuickPick.ignoreFocusOut = true;
+		orgQuickPick.placeholder = 'Organization Name';
+		// TODO: Should this be set to owner?  It kind of makes the fact that you can publish to other
+		// organizations hidden until you clear the value.
+		orgQuickPick.value = owner;
```

Comment:
I think you can make options always show, despite what the input value is. Give that a try.